M2/Macaulay2/packages/Macaulay2Doc/changes.m2

```diff
@@ -36,6 +36,9 @@ document {
      UL {
          LI { "functionality improved",
               UL {
+                   LI { "The ", TO document, " function now accepts three new options: ", TT "Acknowledgement", ", ",
+		       TT "Contributors", ", and ", TT "References", ". The content of each will appear under a subsection
+		       of the same name in the documentation." },
```

Comment:
Mention also that "document" will now accept a string, doing the same thing "doc" would have done.

Reply:
Sure.